packages/autocomplete-core/src/utils/__tests__/isChrome.test.ts

```diff
@@ -0,0 +1,119 @@
+import { isChrome } from '../isChrome';
+
+describe('isChrome', () => {
+  describe('Android', () => {
+    describe('Mobile/tablet with Chrome', () => {
+      test('returns true with a Samsung S9 (Mobile - Chrome) user agent', () => {
+        const ua =
+          'Mozilla/5.0 (Linux; Android 8.0.0; SM-G960F Build/R16NW) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.84 Mobile Safari/537.36';
+
+        expect(isChrome(ua)).toEqual(true);
+      });
+
+      test('returns true with a Samsung Galaxy Tab S3 (Tablet - Chrome) user agent', () => {
```

Comment:
We know we're on Chrome already.

Same for other tests.

```suggestion
      test('returns true with a Samsung Galaxy Tab S3 (tablet) user agent', () => {
```